### DISCUSSION TITLE
xtls-plugin未知插件

### DISCUSSION THREAD

**Discussion by zcluo** [SagerNet/SagerNet#100] [Q&A]
Tcp xtls 报未知插件![Screenshot_2021-05-19-19-16-15-111_io.nekohasekai.sagernet.jpg](https://user-images.githubusercontent.com/642516/118805414-79275680-b8d8-11eb-95be-8ba7c8eb2bce.jpg)

**Comment by zcluo** [comment]
已解决

**Reply by sscstone** [reply]
请问如何解决呢，我也出现这个未知插件
